server/ntb/publish/ntb_nitf.py

```diff
@@ -117,7 +117,7 @@ def _get_filename(self, article):
         example: 2016-08-16_11-07-46_Nyhetstjenesten_Innenriks_ny1-rygge-nedgang.xml
         """
         metadata = {}
-        metadata['date'] = article['versioncreated'].strftime("%Y-%m-%d_%H-%M-%S")
+        metadata['date'] = article['versioncreated'].astimezone().strftime("%Y-%m-%d_%H-%M-%S")
```

Comment:
oh I didn't knew there was this in settings, yes it makes definitely more sense then, I'll update the P.R.